### ISSUE TITLE
Incompatibility with V2 DS3232RTC library

### ISSUE THREAD

**Issue by M5ET** [m1geo/WordClock#21]
The DS3232RTC library has changed significantly - with 1.3.0 the sketch compiles just fine, but as soon as you update to v2 (or 2.0.1), it fails to compile as follows:

```
Compiling sketch...
"Path\\To\\ArduinoData\\packages\\arduino\\tools\\avr-gcc\\7.3.0-atmel3.6.1-arduino7/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -Wno-error=narrowing -MMD -flto -mmcu=atmega328p -DF_CPU=16000000L -DARDUINO=10819 -DARDUINO_AVR_NANO -DARDUINO_ARCH_AVR "-IPath\\To\\ArduinoData\\packages\\arduino\\hardware\\avr\\1.8.6\\cores\\arduino" "-IPath\\To\\ArduinoData\\packages\\arduino\\hardware\\avr\\1.8.6\\variants\\eightanaloginputs" "-IPath\\To\\Arduino\\libraries\\DS3232RTC\\src" "-IPath\\To\\Arduino\\libraries\\Time" "-IPath\\To\\ArduinoData\\packages\\arduino\\hardware\\avr\\1.8.6\\libraries\\Wire\\src" "-IPath\\To\\Arduino\\libraries\\Adafruit_GFX_Library" "-IPath\\To\\Arduino\\libraries\\Adafruit_BusIO" "-IPath\\To\\ArduinoData\\packages\\arduino\\hardware\\avr\\1.8.6\\libraries\\SPI\\src" "-IPath\\To\\Arduino\\libraries\\Adafruit_NeoMatrix" "-IPath\\To\\Arduino\\libraries\\Adafruit_NeoPixel" "-IPath\\To\\ArduinoData\\packages\\arduino\\hardware\\avr\\1.8.6\\libraries\\EEPROM\\src" "-IPath\\To\\Arduino\\libraries\\WordClock_Mapping" "Path\\To\\AppData\\Local\\Temp\\arduino_build_163437\\sketch\\WordClock.ino.cpp" -o "Path\\To\\AppData\\Local\\Temp\\arduino_build_163437\\sketch\\WordClock.ino.cpp.o"
Path\To\Arduino\WordClock-master\WordClock\WordClock.ino: In function 'void setup()':
WordClock:150:19: error: 'RTC' was not declared in this scope
   setSyncProvider(RTC.get);
                   ^~~
Path\To\Arduino\WordClock-master\WordClock\WordClock.ino: In function 'void loop()':
WordClock:202:19: error: 'RTC' was not declared in this scope
         float c = RTC.temperature() / 4.0;
                   ^~~
Path\To\Arduino\WordClock-master\WordClock\WordClock.ino: In function 'void scrollEverything()':
WordClock:218:13: error: 'RTC' was not declared in this scope
   float c = RTC.temperature() / 4.0;
             ^~~
Path\To\Arduino\WordClock-master\WordClock\WordClock.ino: In function 'void setDate()':
WordClock:306:7: error: 'RTC' was not declared in this scope
       RTC.set(t);        // use the time_t value to ensure correct weekday is set
       ^~~
Using library DS3232RTC at version 2.0.1 in folder: Path\To\Arduino\libraries\DS3232RTC 
Using library Time at version 1.6.1 in folder: Path\To\Arduino\libraries\Time 
Using library Wire at version 1.0 in folder: Path\To\ArduinoData\packages\arduino\hardware\avr\1.8.6\libraries\Wire 
Using library Adafruit_GFX_Library at version 1.11.5 in folder: Path\To\Arduino\libraries\Adafruit_GFX_Library 
Using library Adafruit_BusIO at version 1.14.1 in folder: Path\To\Arduino\libraries\Adafruit_BusIO 
Using library SPI at version 1.0 in folder: Path\To\ArduinoData\packages\arduino\hardware\avr\1.8.6\libraries\SPI 
Using library Adafruit_NeoMatrix at version 1.3.0 in folder: Path\To\Arduino\libraries\Adafruit_NeoMatrix 
Using library Adafruit_NeoPixel at version 1.11.0 in folder: Path\To\Arduino\libraries\Adafruit_NeoPixel 
Using library EEPROM at version 2.0 in folder: Path\To\ArduinoData\packages\arduino\hardware\avr\1.8.6\libraries\EEPROM 
Using library WordClock_Mapping in folder: Path\To\Arduino\libraries\WordClock_Mapping (legacy)
exit status 1
'RTC' was not declared in this scope
```

**Comment by m1geo** [comment]
Thanks for the heads up, John. If you do update things for V2.x of the library, please feel free to submit a PR and I'll merge it in.

**Comment by m1geo** [comment]
Fixed by e1c3005
